package.json

```diff
@@ -30,12 +33,14 @@
     "eslint-release": "^3.2.0",
     "mocha": "^9.0.1",
     "opener": "^1.5.2",
-    "rollup": "^2.52.1"
+    "rollup": "^2.52.1",
+    "typescript": "^4.5.5"
   },
   "scripts": {
-    "prepare": "npm run build",
+    "prepare": "npm run build && npm run tsc",
```

Comment:
I think `&& npm run tsc` should be in the `build` script.

Reply:
Sure, done...